integration-cli/docker_cli_events_test.go

```diff
@@ -75,40 +75,6 @@ func (s *DockerSuite) TestEventsUntag(c *check.C) {
 	}
 }
 
-func (s *DockerSuite) TestEventsContainerFailStartDie(c *check.C) {
```

Comment:
If we run the tests with this feature enabled these two will fail because they check that the cmd passed does not exist and it will always exist because we prepend our init.

It will still exit with a 127 but not at start time.  I tried looking through the history to see why these these were added but I couldn't find anything.  

We went over this same discussion in the runc change for 1.11  and I said that if we were to ever solve the zombie problem this is not going to work.  We get the same outcome, the containers fails with 127 exit status but it does not return the error on start.